ppr-ui/src/store/getters/state-getters.ts

```diff
@@ -212,6 +212,20 @@ export const isMhrRegistration = (state: StateIF): boolean => {
     APIRegistrationTypes.MANUFACTURED_HOME_REGISTRATION
 }
 
+/** Is true when all steps are valid as well as staff payment and authorization are valid */
+export const isReviewConfirmValid = (state: StateIF): boolean => {
+  return state.stateModel.mhrValidationState.reviewConfirmValid.authorizationValid &&
+  state.stateModel.mhrValidationState.reviewConfirmValid?.staffPaymentValid &&
```

Comment:
Sorry, might want to specify MHR in the name. 

Although staffPaymentValid still needs to be applied conditionally, it doesn't necessarily need to be done here/now, so all good!

Reply:
Whoops, thx

Reply:
Would the staff Payment check be conditional on a QS account?

Reply:
It would just be conditional on it being Staff user. 
That being said, thats the target release and we haven't developed for non staff as of yet, so no worries. 
No action required here :)